version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "0.1.4"
+ThisBuild / version := "0.1.5"
```

Comment:
let's make it 0.2 since it's not backwards compatible

Reply:
got it.  version updated.